dids/src/main/kotlin/web5/sdk/dids/methods/dht/DidDht.kt

```diff
@@ -136,7 +135,7 @@ public sealed class DidDhtApi(configuration: DidDhtConfiguration) : DidMethod<Di
     val publicKey = keyManager.getPublicKey(keyAlias)
 
     // build DID Document
-    val id = DidDht.getDidIdentifier(publicKey)
+    val didUrl = DidDht.getDidIdentifier(publicKey)
```

Comment:
todo probably should find out what this method returns and call it something other than `didUrl` since it's probably not the full did

Reply:
changed to `didUri`